core/src/main/java/org/apache/carbondata/core/scan/expression/ColumnExpression.java

```diff
@@ -31,12 +32,16 @@
 
   private boolean isDimension;
 
+  private boolean isMeasure;
```

Comment:
same information is available inside `measure`, so please remove it and get it from `measure`